packaging/Dockerfile

```diff
@@ -1,8 +1,14 @@
-FROM registry.opensource.zalan.do/library/alpine-3.13:latest
+ARG BASE_IMAGE=registry.opensource.zalan.do/library/alpine-3.13:latest
+FROM ${BASE_IMAGE}
 LABEL maintainer="Team Teapot @ Zalando SE <team-teapot@zalando.de>"
 RUN apk --no-cache add ca-certificates && update-ca-certificates
 RUN mkdir -p /usr/bin
-ADD skipper eskip webhook routesrv /usr/bin/
+ARG BUILD_FOLDER=build
+ARG TARGETPLATFORM
+ADD ${BUILD_FOLDER}/${TARGETPLATFORM}/skipper \
+    ${BUILD_FOLDER}/${TARGETPLATFORM}/eskip \
+    ${BUILD_FOLDER}/${TARGETPLATFORM}/webhook \
+    ${BUILD_FOLDER}/${TARGETPLATFORM}/routesrv /usr/bin/
```

Comment:
This is not perfectly backwards compatible because of `BUILD_FOLDER=build` instead of `BUILD_FOLDER=.`. However, the corresponding task in the Makefile provides that.

Changing the default to `.` would require the majority of `docker build` invocations to overwrite it. Having the default like this only requires one task to overwrite it.